flixel/addons/editors/tiled/TiledMap.hx

```diff
@@ -31,30 +31,29 @@ class TiledMap
 	
 	public var properties:TiledPropertySet;
 	
+	public var tilesets: Map<String, TiledTileSet>;
+	
+	public var layers:Array<TiledLayer>;
+	
 	// Add a "noload" property to your Map Properties.
 	// Add comma separated values of tilesets, layers, or object names.
 	// These will not be loaded.
 	private var noLoadHash:Map<String, Bool>;
+	private var layerMap:Map<String, TiledLayer>;
 	
-	// Use hash, we don't care about order
-	public var tilesets: Map<String, TiledTileSet>;
-	// Use array to preserve load order
-	public var layers:Array<TiledLayer>;
-	public var objectGroups:Array<TiledObjectGroup>;
-	
-	public function new(Data:Dynamic)
+	public function new(data:Dynamic)
```

Comment:
Not a fan of the `Dynamic` here, we should update this to use an asset abstract like core flixel does for images, sounds etc, something like:

`typedef FlxTiledAsset = OneOfTwo<String, Xml>;`

That way, we don't need the else-case that throws an error either.

I guess we would have the abstracts in `FlxAssets` non-private...